nb/projects/mold/hex.md

```diff
@@ -13,14 +13,8 @@ const assembled = Hexagon(50, 50, 50)
 ![Image](hex.md.0.png)
 
 ```JavaScript
-const disassembled = Hexagon(50, 50, 50)
-  .eachEdge(
-    (e, l) =>
-      Box([0, l], [0, 5], [0, -1])
-        .join(ArcX([0, l], [4, 7.0], 3).hasAngle(0 / 4, 2 / 4))
-        .to(e),
-    (f, e) => f.e(1).join(e).to(XY())
-  )
+const disassembled = assembled
```

Comment:
Well this is simpler :)

Reply:
Yeah, getting there.

Reply:
Aren't you supposed to be on vacation? :)